```
Change Dir: 'D:/b/unarr/x64-windows-dbg'

Run Build Command(s): D:/downloads/tools/ninja/1.13.1-windows/ninja.exe -v -v -j33 install
[1/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\common\crc32.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\common\crc32.c
[2/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\zip\inflate.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\zip\inflate.c
[3/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\_7z\_7z.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\_7z\_7z.c
[4/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\common\conv.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\common\conv.c
[5/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\common\unarr.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\common\unarr.c
[6/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\tar\tar.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\tar\tar.c
[7/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\tar\parse-tar.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\tar\parse-tar.c
[8/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\rar\rar.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\rar\rar.c
[9/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\rar\filter-rar.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\rar\filter-rar.c
[10/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\rar\huffman-rar.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\rar\huffman-rar.c
[11/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\rar\parse-rar.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\rar\parse-rar.c
[12/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\zip\parse-zip.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\zip\parse-zip.c
[13/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\rar\rarvm.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\rar\rarvm.c
[14/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\rar\uncompress-rar.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\rar\uncompress-rar.c
[15/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\zip\uncompress-zip.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\zip\uncompress-zip.c
D:\b\unarr\src\v1.1.1-74767a1cdd.clean\zip\uncompress-zip.c(238): warning C4267: 'return': conversion from 'size_t' to 'uint32_t', possible loss of data
D:\b\unarr\src\v1.1.1-74767a1cdd.clean\zip\uncompress-zip.c(273): warning C4267: 'return': conversion from 'size_t' to 'uint32_t', possible loss of data
[16/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\zip\zip.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\zip\zip.c
[17/19] C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\cl.exe   -DHAVE_7Z -DHAVE_BZIP2 -DHAVE_LIBLZMA -DHAVE_ZLIB -DUNARR_EXPORT_SYMBOLS -DUNARR_IS_SHARED_LIBRARY -DUNICODE -DUSE_ZLIB_CRC -DZ7_PPMD_SUPPORT -D_CRT_SECURE_NO_WARNINGS -D_UNICODE -ID:\b\unarr\src\v1.1.1-74767a1cdd.clean -external:ID:\installed\x64-windows\include\7zip\CPP -external:ID:\installed\x64-windows\include\7zip\C -external:ID:\installed\x64-windows\include -external:W0 /nologo /DWIN32 /D_WINDOWS /utf-8 /MP  /MDd /Z7 /Ob0 /Od /RTC1  /W3 /showIncludes /FoCMakeFiles\unarr.dir\common\stream.c.obj /FdCMakeFiles\unarr.dir\ /FS -c D:\b\unarr\src\v1.1.1-74767a1cdd.clean\common\stream.c
[18/19] C:\Windows\system32\cmd.exe /C "cd . && D:\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386\bin\cmake.exe -E vs_link_dll --intdir=CMakeFiles\unarr.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\link.exe  CMakeFiles\unarr.dir\_7z\_7z.c.obj CMakeFiles\unarr.dir\common\conv.c.obj CMakeFiles\unarr.dir\common\crc32.c.obj CMakeFiles\unarr.dir\common\stream.c.obj CMakeFiles\unarr.dir\common\unarr.c.obj CMakeFiles\unarr.dir\rar\filter-rar.c.obj CMakeFiles\unarr.dir\rar\uncompress-rar.c.obj CMakeFiles\unarr.dir\rar\huffman-rar.c.obj CMakeFiles\unarr.dir\rar\rar.c.obj CMakeFiles\unarr.dir\rar\rarvm.c.obj CMakeFiles\unarr.dir\rar\parse-rar.c.obj CMakeFiles\unarr.dir\tar\parse-tar.c.obj CMakeFiles\unarr.dir\tar\tar.c.obj CMakeFiles\unarr.dir\zip\inflate.c.obj CMakeFiles\unarr.dir\zip\parse-zip.c.obj CMakeFiles\unarr.dir\zip\uncompress-zip.c.obj CMakeFiles\unarr.dir\zip\zip.c.obj  /out:unarr.dll /implib:unarr.lib /pdb:unarr.pdb /dll /version:1.1 /machine:x64 /nologo    /debug /INCREMENTAL  D:\installed\x64-windows\debug\lib\7zip.lib  D:\installed\x64-windows\debug\lib\bz2d.lib  D:\installed\x64-windows\debug\lib\lzma.lib  D:\installed\x64-windows\debug\lib\zlibd.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
FAILED: [code=4294967295] unarr.dll unarr.lib 
C:\Windows\system32\cmd.exe /C "cd . && D:\downloads\tools\cmake-3.30.1-windows\cmake-3.30.1-windows-i386\bin\cmake.exe -E vs_link_dll --intdir=CMakeFiles\unarr.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100261~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\link.exe  CMakeFiles\unarr.dir\_7z\_7z.c.obj CMakeFiles\unarr.dir\common\conv.c.obj CMakeFiles\unarr.dir\common\crc32.c.obj CMakeFiles\unarr.dir\common\stream.c.obj CMakeFiles\unarr.dir\common\unarr.c.obj CMakeFiles\unarr.dir\rar\filter-rar.c.obj CMakeFiles\unarr.dir\rar\uncompress-rar.c.obj CMakeFiles\unarr.dir\rar\huffman-rar.c.obj CMakeFiles\unarr.dir\rar\rar.c.obj CMakeFiles\unarr.dir\rar\rarvm.c.obj CMakeFiles\unarr.dir\rar\parse-rar.c.obj CMakeFiles\unarr.dir\tar\parse-tar.c.obj CMakeFiles\unarr.dir\tar\tar.c.obj CMakeFiles\unarr.dir\zip\inflate.c.obj CMakeFiles\unarr.dir\zip\parse-zip.c.obj CMakeFiles\unarr.dir\zip\uncompress-zip.c.obj CMakeFiles\unarr.dir\zip\zip.c.obj  /out:unarr.dll /implib:unarr.lib /pdb:unarr.pdb /dll /version:1.1 /machine:x64 /nologo    /debug /INCREMENTAL  D:\installed\x64-windows\debug\lib\7zip.lib  D:\installed\x64-windows\debug\lib\bz2d.lib  D:\installed\x64-windows\debug\lib\lzma.lib  D:\installed\x64-windows\debug\lib\zlibd.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
LINK Pass 1: command "C:\PROGRA~1\MICROS~1\2022\ENTERP~1\VC\Tools\MSVC\1444~1.352\bin\Hostx64\x64\link.exe CMakeFiles\unarr.dir\_7z\_7z.c.obj CMakeFiles\unarr.dir\common\conv.c.obj CMakeFiles\unarr.dir\common\crc32.c.obj CMakeFiles\unarr.dir\common\stream.c.obj CMakeFiles\unarr.dir\common\unarr.c.obj CMakeFiles\unarr.dir\rar\filter-rar.c.obj CMakeFiles\unarr.dir\rar\uncompress-rar.c.obj CMakeFiles\unarr.dir\rar\huffman-rar.c.obj CMakeFiles\unarr.dir\rar\rar.c.obj CMakeFiles\unarr.dir\rar\rarvm.c.obj CMakeFiles\unarr.dir\rar\parse-rar.c.obj CMakeFiles\unarr.dir\tar\parse-tar.c.obj CMakeFiles\unarr.dir\tar\tar.c.obj CMakeFiles\unarr.dir\zip\inflate.c.obj CMakeFiles\unarr.dir\zip\parse-zip.c.obj CMakeFiles\unarr.dir\zip\uncompress-zip.c.obj CMakeFiles\unarr.dir\zip\zip.c.obj /out:unarr.dll /implib:unarr.lib /pdb:unarr.pdb /dll /version:1.1 /machine:x64 /nologo /debug /INCREMENTAL D:\installed\x64-windows\debug\lib\7zip.lib D:\installed\x64-windows\debug\lib\bz2d.lib D:\installed\x64-windows\debug\lib\lzma.lib D:\installed\x64-windows\debug\lib\zlibd.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST /MANIFESTFILE:CMakeFiles\unarr.dir/intermediate.manifest CMakeFiles\unarr.dir/manifest.res" failed (exit code 1120) with the following output:
   Creating library unarr.lib and object unarr.exp
_7z.c.obj : error LNK2019: unresolved external symbol LookToRead2_CreateVTable referenced in function ar_open_7z_archive
_7z.c.obj : error LNK2019: unresolved external symbol SzArEx_Init referenced in function ar_open_7z_archive
_7z.c.obj : error LNK2019: unresolved external symbol SzArEx_Free referenced in function _7z_close
_7z.c.obj : error LNK2019: unresolved external symbol SzArEx_Extract referenced in function _7z_uncompress
_7z.c.obj : error LNK2019: unresolved external symbol SzArEx_Open referenced in function ar_open_7z_archive
uncompress-rar.c.obj : error LNK2019: unresolved external symbol Ppmd7_Construct referenced in function rar_parse_codes
uncompress-rar.c.obj : error LNK2019: unresolved external symbol Ppmd7_Alloc referenced in function rar_parse_codes
uncompress-rar.c.obj : error LNK2019: unresolved external symbol Ppmd7_Free referenced in function rar_clear_uncompress
uncompress-rar.c.obj : error LNK2019: unresolved external symbol Ppmd7_Init referenced in function rar_parse_codes
uncompress-rar.c.obj : error LNK2019: unresolved external symbol Ppmd7a_RangeDec_Init referenced in function rar_parse_codes
uncompress-rar.c.obj : error LNK2019: unresolved external symbol Ppmd7a_DecodeSymbol referenced in function rar_decode_ppmd7_symbol
uncompress-zip.c.obj : error LNK2019: unresolved external symbol Ppmd8_Construct referenced in function zip_init_uncompress_ppmd
uncompress-zip.c.obj : error LNK2019: unresolved external symbol Ppmd8_Alloc referenced in function zip_uncompress_data_ppmd
uncompress-zip.c.obj : error LNK2019: unresolved external symbol Ppmd8_Free referenced in function zip_clear_uncompress_ppmd
uncompress-zip.c.obj : error LNK2019: unresolved external symbol Ppmd8_Init referenced in function zip_uncompress_data_ppmd
uncompress-zip.c.obj : error LNK2019: unresolved external symbol Ppmd8_Init_RangeDec referenced in function zip_uncompress_data_ppmd
uncompress-zip.c.obj : error LNK2019: unresolved external symbol Ppmd8_DecodeSymbol referenced in function zip_uncompress_data_ppmd
unarr.dll : fatal error LNK1120: 17 unresolved externals
ninja: build stopped: subcommand failed.
```

I think nor unarr nor 7zip intend to support Windows OS shared library build.

--- a/ports/unarr/portfile.cmake
+++ b/ports/unarr/portfile.cmake
@@ -1,3 +1,7 @@
+if (VCPKG_TARGET_IS_WINDOWS)
+    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
+endif()
+
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO selmf/unarr

--- a/ports/unarr/portfile.cmake
+++ b/ports/unarr/portfile.cmake
@@ -1,7 +1,3 @@
-if (VCPKG_TARGET_IS_WINDOWS)
-    vcpkg_check_linkage(ONLY_STATIC_LIBRARY)
-endif()
-
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO selmf/unarr

--- a/ports/unarr/vcpkg.json
+++ b/ports/unarr/vcpkg.json
@@ -4,6 +4,7 @@
   "description": "A decompression library for rar, tar, zip and 7z archives.",
   "homepage": "https://github.com/selmf/unarr",
   "license": "LGPL-3.0-only",
+  "supports": "!(windows & !static)",
   "dependencies": [
     "7zip",
     "bzip2",

--- a/versions/u-/unarr.json
+++ b/versions/u-/unarr.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "3d49fd543b180134e24f760872d774367215df3a",
+      "git-tree": "5300e67f7b0e4aed9f3f49176ef8e25ba09a11f9",
       "version": "1.1.1",
       "port-version": 0
     }

